app/models/conference.rb

```diff
@@ -4,7 +4,9 @@ class Conference < ActiveRecord::Base
 
   has_many :attendances
   has_many :attendees, through: :attendances, source: :user
+
   validates :name, :round, presence: true
+  validate :chronology # custom validator
```

Comment:
Ah, I see what you mean. The mean difference I see, though, is that `name` is a model attribute whereas the custom validation we're adding here isn't. But I'm not super invested into the naming of this helper so you decide! :smile: